pkg/services/folder/foldertest/foldertest.go

```diff
@@ -31,8 +31,7 @@ func (s *FakeService) GetFolderByTitle(ctx context.Context, user *user.SignedInU
 func (s *FakeService) CreateFolder(ctx context.Context, cmd *folder.CreateFolderCommand) (*models.Folder, error) {
 	return s.ExpectedFolder, s.ExpectedError
 }
-func (s *FakeService) UpdateFolder(ctx context.Context, user *user.SignedInUser, orgID int64, existingUid string, cmd *models.UpdateFolderCommand) error {
-	cmd.Result = s.ExpectedFolder
```

Comment:
is it ok to remove this? our `folder.UpdateFolderCommand` doesn't have Result, and we dont return anything from `UpdateFolder`either